### DISCUSSION TITLE
Extremely low volume on Android phone.

### DISCUSSION THREAD

**Discussion by nbanderso** [mcguirepr89/BirdNET-Pi#871] [General]
For some reason when I am playing back bird detections on my Android phone (Galaxy S23+), the volume is incredibly low.  I have to turn the media volume up to max and hold the phone right to my ear to hear it.

The same detections on an apple device are at a reasonably volume.  I also tried different browsers on the same android phone and it's still low.

Any ideas?

**Comment by ehpersonal38** [comment]
You might find this wiki article helpful for increasing the volume of your microphone for future recordings: https://github.com/mcguirepr89/BirdNET-Pi/wiki/Adjusting-your-sound-card
